src/components/Reports/AdditionalSalaryRequest/Shared/AdditionalSalaryRequestContext.tsx

```diff
@@ -116,8 +117,11 @@ export const AdditionalSalaryRequestProvider: React.FC<Props> = ({
         autoPurchase: createCurrencyValidation('Auto Purchase'), // Max will eventually be a constant, no determined value yet
         reimbursableExpenses: createCurrencyValidation('Reimbursable Expenses'),
         defaultPercentage: yup.boolean(),
+        telephoneNumber: yup
```

Comment:
All of the field names passed to `createCurrencyValidation` should be localized too.